src/i18n/sv.json

```diff
@@ -0,0 +1,160 @@
+{
+  "appName": "Profil",
+  "berthErrors": {
+    "close": "Stänga",
+    "explanation": "Du kan inte radera din information eftersom Båtplatser-tjänsten behöver din faktureringsinformation. Kontakta Båtplatsers kundservice för att säga upp ditt kontrakt. När det är gjort kan du försöka igen.",
+    "title": "Operationen kan inte slutföras."
+  },
```

Comment:
This file contains some texts that probably aren't used in this demo.

Reply:
File removed.